src/App/Pages/Vault/CipherAddEditPageViewModel.cs

```diff
@@ -307,6 +307,7 @@ public bool HasCollections
         public bool PasswordPrompt => Cipher.Reprompt != CipherRepromptType.None;
         public string PasswordVisibilityAccessibilityText => ShowPassword ? AppResources.PasswordIsVisibleTapToHide : AppResources.PasswordIsNotVisibleTapToShow;
         public bool HasTotpValue => IsLogin && !string.IsNullOrEmpty(Cipher?.Login?.Totp);
+        public bool AllowTotpCopy => HasTotpValue && (Cipher?.ViewPassword ?? false);
```

Comment:
:art: No need for parenthesis and IMO equal to true is more readable.
```suggestion
        public bool AllowTotpCopy => HasTotpValue && Cipher?.ViewPassword == true;
```
Aside from this tiny fix, should we raise property changed when the `Cipher` property changes? I ask because if `Cipher` starts with `null` value then if `AllowTotpCopy` gets evaluated will be `false` and when `Cipher` is loaded currently it won't trigger `AllowTotpCopy` property changed so there will be an inconsistency. Thoughts?